### DISCUSSION TITLE
remove the versionless schema redirect on the website

### DISCUSSION THREAD

**Discussion by karenetheridge** [json-schema-org/community#160] [General]
context: https://json-schema.slack.com/archives/C5CF75URH/p1647254190574629

**Comment by gregsdennis** [comment]
Should it be removed or version-locked and documented as deprecated? I suspect removing it could break a lot of people.

**Reply by karenetheridge** [reply]
There's already a redirect that's been in place for.. I don't know how long, but years. It's still being used but it's invalid, so as I said in the slack thread I think it's time to bring the hammer down. "deprecated" is not even the situation we're in right now -- this `$schema` URI is outright *not supported* by the specification, and since anyone using it will be getting the draft2020-12 version of the file, it won't even work with their implementation anyway (once an implementation sees `"$schema": "https://json-schema.org/draft/2020-12/schema"` it should be stopping with a "this is a version of the spec I don't know how to interpret") -- so they're already broken, in multiple ways.

There is no way that using this URI can be giving users any sort of useful result. This change would just be making the breakage more visible, and accelerate the process of getting it fixed.

If it were possible to get the access logs for this file we could attempt to contact the implementations via the User-Agent headers and warn them. Otherwise, I think we should JFDI.

**Reply by jdesrosiers** [reply]
What's the last release where the "latest" URI was supported by the specification? Maybe we should version-lock at that version.

**Comment by Relequestual** [comment]
Thread of links where a versionless URI is being used...

**Reply by Relequestual** [reply]
- https://github.com/hubmapconsortium/ingest-pipeline/blob/d8f9fbe662fc616d91c6c2f766a9e16acf2babb6/src/ingest-pipeline/schemata/enumerated_types_schema.yml

**Reply by karenetheridge** [reply]
could you also link to reported issues as they are created, to reduce duplicated effort and enable tracking of fix progress. (https://github.com/hubmapconsortium/ingest-pipeline/issues/535)
